### ISSUE TITLE
Bedrock simulation support

### ISSUE THREAD

**Issue by ghost** [GrimAnticheat/Grim#1065]
### Feature description

Rudimentary rough checking for bedrock. would be good. i.e bedrock playerrs shouldn't completely bypass and be allowed to fly super fast, there should be approximate checking, as bedrock movement is approximately equal to java.

**Comment by MWHunter** [comment]
No.  This is too much work.  The only way I'm touching bedrock is if I'm sponsored by a large server to do so, which there doesn't seem to be any large server with Geyser support.

**Comment by MachineBreaker** [comment]
> ### Feature description
> Rudimentary rough checking for bedrock. would be good. i.e bedrock playerrs shouldn't completely bypass and be allowed to fly super fast, there should be approximate checking, as bedrock movement is approximately equal to java.

Your welcome to contribute

**Comment by caoli5288** [comment]
> No. This is too much work. The only way I'm touching bedrock is if I'm sponsored by a large server to do so, which there doesn't seem to be any large server with Geyser support.

I want to know how many player's servers or how much sponsorship will get you started working?
